lib/rules/no-warning-comments.js

```diff
@@ -141,12 +143,28 @@ module.exports = {
 
             const matches = commentContainsWarningTerm(node.value);
 
-            matches.forEach(matchedTerm => {
+            matches.forEach(({ type, comment }) => {
+                let len = 0;
+                const comments = [];
+                let truncated = false;
+
+                comment
+                    .split(/\s/u).forEach(c => {
```

Comment:
Maybe we could do `/\s+/u` instead, to turn a sequence of whitespace into just one space?

Multiline comments will often have some indentation after a linebreak, so it might be nice to not show all that blank space in the message?

Reply:
cool